src/decision_model.jl

```diff
@@ -1,11 +1,16 @@
 using JuMP
 
-function decision_variable(model::Model, S::States, d::Node, I_d::Vector{Node}, base_name::String="")
+function decision_variable(model::Model, S::States, d::Node, I_d::Vector{Node}, names::Bool, base_name::String="")
     # Create decision variables.
     dims = S[[I_d; d]]
     z_d = Array{VariableRef}(undef, dims...)
     for s in paths(dims)
-        z_d[s...] = @variable(model, binary=true, base_name=base_name)
+        if names == true
+            name = join([base_name, s...], "_")
+            z_d[s...] = @variable(model, binary=true, base_name=name)
+        else
+            z_d[s...] = @variable(model, binary=true, base_name=base_name)
```

Comment:
This might not feel logical for the user: names==false results in the variables using base_name. I think the else-branch should be `z_d[s...] = @variable(model, binary=true)`